docs/sources/operator/architecture.md

```diff
@@ -132,3 +132,69 @@ Two labels are added by default to every metric:
 
 The shard number is not added as a label, as sharding is designed to be
 transparent on the receiver end.
+
+## Enabling sharding and replication
+
+To enable sharding and replication, you need to set the `shards` and `replicas` properties in the Grafana Agent configuration file. For example, the following configuration file would shard the data into 3 shards and replicate each shard to 2 other Grafana Agent instances:
+
+```
+shards: 3
+replicas: 2
+```
+
+You can also enable sharding and replication by setting the `shards` and `replicas` arguments when you start the Grafana Agent. 
+
+### Examples
+
+Here are some examples of how to enable sharding and replication in a Kubernetes environment:
+
+* To shard the data into 3 shards and replicate each shard to 2 other Grafana Agent instances, you would use the following deployment manifest:
+
+```
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: grafana-agent
+spec:
+  replicas: 3
+  selector:
+    matchLabels:
+      app: grafana-agent
+  template:
+    metadata:
+      labels:
+        app: grafana-agent
+    spec:
+      containers:
+      - name: grafana-agent
+        image: grafana/agent:latest
+        args:
+        - "--shards=3"
+        - "--replicas=2"
+```
```

Comment:
For alignment, this (and the following example code block) should be indented to align with the first char of the bullet the precedes it. This will align the code block with the bullet text.